README.md

```diff
@@ -441,6 +442,17 @@ When the code is ready, submit a Pull Request.
 See also Trino's [guidelines](https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md).
 Most of them also apply to code in trino-python-client.
 
+### `pre-commit` checks
+
+Code is automatically checked on commit by a [pre-commit](https://pre-commit.com/) git hook.
+
+Following checks are performed:
+
+- [`flake8`](https://flake8.pycqa.org/en/latest/) for code linting
```

Comment:
linting should run on CI for now. We don't want to prevent from submitting PRs or pushing changes just when experimenting or when one needs help to fix whatever is being complained about.

Reply:
That can be disabled by `git commit . -m 'quick fix' --no-verify`. We can add this to the docs as well. 

Note that in general only mypy might stop you. black and isort just autoformat your files.

IMHO in general it will be better to always run it on commit to prevent contributors from the experience of failing the build for easily fixable problems.

WDYT?

Reply:
I agreed with black and isort. I just meant to keep flake8 and mypy outside of pre-commit for now. We can iterate.